src/util/Permissions.js

```diff
@@ -10,19 +10,25 @@ const BitField = require('./BitField');
  */
 class Permissions extends BitField {
   /**
-   * @name Permissions
-   * @kind constructor
-   * @memberof Permissions
    * @param {PermissionResolvable} [bits=0] Bit(s) to read from
    */
+  constructor(bits = 0n) {
+    super(bits);
```

Comment:
Might be a good idea to enforce the super call with a `BigInt(bits)` in case someone does `new Permissions(0)`